prover/circuit.go

```diff
@@ -160,12 +176,16 @@ func (circuit *MbuCircuit) Define(api frontend.API) error {
 		currentIndex := api.Add(circuit.StartIndex, i)
 		currentPath := api.ToBinary(currentIndex, circuit.Depth)
 
+		// len(circuit.MerkleProofs[i]) === circuit.Depth
```

Comment:
commented out code

Reply:
I added these as comments for reference. Should I delete?